xtra-libp2p/src/utils.rs

```diff
@@ -0,0 +1,32 @@
+use anyhow::ensure;
+use anyhow::Context;
+use anyhow::Result;
+use std::net::SocketAddr;
+
+use libp2p_core::Multiaddr;
+use libp2p_core::PeerId;
+
+/// Creates MultiAddr from SocketAddr and PeerId
+pub fn create_connect_tcp_multiaddr(
```

Comment:
I reckon these should live outside of `xtra-libp2p`. This utils are super specific to your usecase and having them in here will create friction if you ever want to move this crate into its own repository and release it on crates.io.

Reply:
why would this create friction? they don't  bring any dependencies (`SocketAddr` is std lib after all)

Reply:
I'll have a think about better placement as we keep going. It's still early days.